packages/notification/src/lit/renderer-directives.d.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { TemplateResult } from 'lit';
+import { DirectiveResult } from 'lit/directive.js';
+import { LitRendererDirective } from '@vaadin/lit-renderer';
```

Comment:
The `@vaadin/lit-renderer'` dependency is missing in the `package.json`, please add it.

Reply:
Good catch, fixed.